gcp-data-drive/bq.go

```diff
@@ -75,7 +75,7 @@ func (b *bqDataPlatform) close() error {
 	return nil
 }
 
-// getBQInterface creates and populates the Bigquery platform client requirementa and returns
+// getBQInterface creates and populates the Bigquery platform client requirements and returns
 // a usable getData() method.
```

Comment:
Might need to change more of this. Perhaps

```suggestion
// getBQInterface creates a BigQuery client and returns a TODO that gets data from BigQuery.
```

I don't remember the name of the interface `*bqDataPlatform` implements -- that's what I mean by the TODO. This doesn't return a `getData` method directly.

Reply:
I corrected with some wording I think is better.